src/coreclr/jit/emit.cpp

```diff
@@ -4102,6 +4117,213 @@ void emitter::emitDispCommentForHandle(size_t handle, GenTreeFlags flag)
 #endif // DEBUG
 }
 
+//****************************************************************************
+// emitRemoveJumpToNextInst:  Checks all jumps in the jump list to see if they are
+//   unconditional jumps generated at the end of BB_ALWAYS basic blocks and whether
+//   they are now a jump to the next instruction, this can happen when the following
+//   basic block emitted no instructions
+//
+// Returns:
+//   true if any jumps have been removed, otherwise returns false
```

Comment:
```suggestion
// emitRemoveJumpToNextInst:  Checks all jumps in the jump list to see if they are
//   unconditional jumps generated at the end of BBJ_ALWAYS basic blocks and whether
//   they are now a jump to the next instruction. This can happen when the following
//   basic block emitted no instructions.
```

Reply:
Done.